src/App.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import { BrowserRouter as Router, Route } from 'react-router-dom'
 import { HomePage } from './routes/HomePage'
+import { RecipeDetails } from './routes/RecipeDetails'
+
```

Comment:
Let's keep it clean 😛 
```suggestion
import { RecipeDetails } from './routes/RecipeDetails'
```